docs/guides/opentracing-jaeger.md

```diff
@@ -0,0 +1,54 @@
+---
+id: opentracing-jaeger
+title: OpenTracing with Jaeger
+---
+
+Saleor has an implementation of OpenTracing used for tracing performance of the application. Jaeger is the currently implemented provider which allows to recap whats collected through dedicated web interface.
+
+Jaeger web interface helps you see through particular request, check how many queries it executes underneath, what these queries are (both SQL and GraphQL) and how much time each particular part of the process takes, all on a timeline of a single request.
```

Comment:
```suggestion
Jaeger web interface helps you see through particular requests, such as checking how many queries it executes underneath, what these queries are (both SQL and GraphQL), how much time each particular part of the process takes, all on a timeline of a single request and much more.
```

This is not limited to only those few things, the tracing is still a work in progress, more will come and it will improve 😄 On Jaeger you can see a map/chart as well, which is a neat and useful feature, not only the segments themselves!